tests/SideBySide/AppConfig.cs

```diff
@@ -19,18 +19,17 @@ public static class AppConfig
 				["Data:SupportsJson"] = "false",
 			};
 
-		private static string CodeRootPath = GetCodeRootPath();
+		public static string CertsPath => Config.GetValue<string>("Data:CertificatesPath")
```

Comment:
Are we sure this handles the directory separator conventions? Looks like the travis build is failing again now, for possibly the same reason as before.

From what I saw only the CertsPath is ever concateting with something else using Path.Combine. TestData doesn't have this problem.

Add the following back?
```
			// normalize path to whatever the system prefers
			.Replace('\\', Path.DirectorySeparatorChar)
			.Replace('/', Path.DirectorySeparatorChar);
```